recipeapp/src/App.js

```diff
@@ -1,25 +1,13 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Recipes from '../src/Recipes'
```

Comment:
Given the suggested changes in `src/Recipes/index.js`, this could then be shortened to
```JavaScript
import Recipes from './Recipes'
```